corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -735,6 +809,24 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         return this.stylesContaining(pattern).length > 0;
     };
 
+    Question.prototype.setWidths = function () {
+        const columnWidth = Question.calculateColumnWidthForPerRowStyle(this.style)
+
+        this.controlWidth = columnWidth === constants.GRID_COLUMNS ? constants.CONTROL_WIDTH : constants.FULL_WIDTH;
```

Comment:
I don't think so since it's being called in the constructor, so `this` would point to the instance. But i know there's some weird thing with scope of `this`. What was the problem you ran into recently?

Reply:
I was setting up a util function as a screen resize listener that I wanted to run a callback function in the place it was called from in the same `this` context. https://github.com/dimagi/commcare-hq/pull/33330/commits/e00ddd200b6eee173a37128f0829bef731408430 Arrow functions ended up saving me a lot of grief, since they keep the same context.

Reply:
oh interesting... the callback seems to make it tricky, but it shouldn't be a problem with how it's used here.